edits to gpu_jobs.md (#340)
CHTC/chtc-website-source: uw-research-computing/gpu-jobs.md

    <th>Number of Servers</th>
    <th>Names</th>
    <th>GPUs / Server</th>

>     <th>GPU Type/CUDADeviceName</th>

Reviewer: ```suggestion
    <th>GPU Type (<code>CUDADeviceName</code>)</th>
```

>     <th>GPU Type(<code>CUDADeviceName</code>)</th>

Reviewer 2: ```suggestion
    <th>GPU Type (<code>CUDADeviceName</code>)</th>
```

One more minor spacing change to help with the line wrapping.  Everything else looks good to me.

>     <th>CUDACapability</th>
>     <th>HasCHTCStaging</th>
>     <th>CUDADriverVersion</th>

Reviewer: ```suggestion
    <th><code>CUDACapability</code></th>
    <th><code>HasCHTCStaging</code></th>
    <th>Max <code>CUDADriverVersion</code></th>
```

	code so that it can run across GPU types and without needing the
	latest version of CUDA.

- **Specify Multiple Requirements (optional)**: Multiple requirements can be specified by using && statements:
	```

> 	requirements = (CUDACapability == 7.5) && (CUDADeviceName = "GeForce RTX 2080 Ti")

Reviewer 2: What do you think about changing this example to require some min or max CUDA capability and some minimum amount of GPU RAM?  We generally encourage users to not request a specific type of GPU because that can constrain where their jobs can run more than is necessary.

Author: Sounds good! I checked using condor_status available attributes for specifying GPU memory. CUDAGlobalMemoryMb is an attribute that can be specified I can push changes.

	```
	{:.sub}

> 	Ensure all specified requirements match the attributes of the machine in interest. HTCondor matches jobs to gpus that match all specified requirements. Otherwise, the jobs will sit idle indefinitely

Reviewer: ```suggestion
	Ensure all specified requirements match the attributes of the GPU/server of interest. HTCondor matches jobs to GPUs that match all specified requirements. Otherwise, the jobs will sit idle indefinitely.
```
